sdv/constraints/tabular.py

```diff
@@ -86,21 +90,29 @@ def _fit(self, table_data):
 
         The fit process consists on:
 
-            - Finding a separtor that works for the
+            - Finding a separator that works for the
               current data by iteratively adding `#` to it.
             - Generating the joint column name by concatenating
               the names of ``self._columns`` with the separator.
+            - Generating a mapping of the unique combinations
+              to a unique identifier.
 
         Args:
             table_data (pandas.DataFrame):
                 Table data.
         """
         self._separator = '#'
-        while not self._valid_separator(table_data, self._separator, self._columns):
```

Comment:
Is this method completely removed now?

Reply:
Yeah, it's only on the base Constraint class now.